components/develop/modules/ROOT/pages/index.adoc

```diff
@@ -2,32 +2,28 @@
 
 == Quick Start
 
-Already an Ethereum developer? Use all your usual tools and deployment methods (Truffle, Hardhat, Remix, Web3) and get your own SKALE Chain RPC Endpoint. Yes - an entire RPC endpoint for your dApp, end-users, and ecosystem.
+Already an Ethereum developer? Use all your usual tools and deployment methods (Truffle, Hardhat, Remix, Web3) and get a SKALE Chain RPC Endpoint. 
 
 === Get a SKALE Endpoint
 
-There are several types of SKALE RPC endpoints to get depending on your development stage: development, network testing, and mainnet.
+For testing, you can use the SKALE public testnet which has 15 chains. Most of these chains are dedicated to other dApps and hubs, except `staging-fast-active-bellatrix` which is provided for general public testing.
 
-**🧑‍💻 For quick development**, a single-node chain is sufficient and recommended to build and test your dApp on a SKALE Chain with IMA and Filestorage. 
+[cols="h,m"]
+|===
+|Chain Name: |staging-fast-active-bellatrix
+|RPC (https):| \https://staging-v3.skalenodes.com/v1/staging-fast-active-bellatrix
+|RPC (wss): | wss://staging-v3.skalenodes.com/v1/ws/staging-fast-active-bellatrix
+|Chain ID: | 0x50877ed6
+|Chain name hash: | 0x0000050877ed6397218923b67054bfcfd054c0012d0682fe896667859b4dede9
+|sFUEL faucet: | \https://sfuel.skale.network/staging/
+|===
 
-.Local and IMA-Bridge development environment
-* ⛓ SKALE Chain + IMA Bridge: xref:ima-sdk.adoc[Use the SDK instructions here].
-
-**✅ For network and load testing**, a multi-node chain is recommended.
-
-.SKALE Dappnet-Rinkeby development environments
-* Request a multi-node Dappnet SKALE Chain with IMA connected to Ethereum Rinkeby
-* https://skale.network/innovators-signup[Request to join the SKALE Innovator Program] to access a multi-node Dappnet SKALE Chain
-
-**🚀 For production ready dApps**, SKALE Mainnet Chains are currently available to select dApps, and will be available on a rolling basis.
-
-.SKALE Mainnet
-* Receive a grant to provision a 16-node SKALE Chain on Mainnet
-* (Soon) Stake SKL to provision a 16-node SKALE Chain on Mainnet
+[NOTE]
+The public chain provided above requires sFUEL to deploy (faucet: https://sfuel.skale.network/staging/). This is unlike other SKALE chains which require both deployer access and sFUEL to deploy.
```

Comment:
Suggestion, Not Required: The public chain provided above uses sFUEL ("gas token") for users to deploy contracts and transaction on the chain. This is unlike other SKALE chains which require both deployer access and sFUEL to deploy.